src/components/CioQuiz/index.tsx

```diff
@@ -87,7 +95,12 @@ export default function CioQuiz(props: IQuizProps) {
           {state.quiz.results || state.quiz.skipToResults ? (
             <ResultContainer options={resultsPageOptions} />
           ) : (
-            state.quiz.currentQuestion && <QuizQuestions />
+            state.quiz.currentQuestion && (
+              <>
+                <QuizQuestions />
+                <ControlBar ctaButtonText={questionData?.cta_text || undefined} />
```

Comment:
`questionData?.cta_text || undefined`

could be simplified to: `questionData?.cta_text`

right?

Reply:
Typescript gets really mad because that returns null and null is not undefined